src/OctoshiftCLI.Tests/AdoApiTests.cs

```diff
@@ -37,5 +42,701 @@ public async void Get_User_Id_Invalid_Response()
             using var sut = new AdoApi(mockClient.Object);
             await Assert.ThrowsAsync<InvalidDataException>(async () => await sut.GetUserId());
         }
+
+        [Fact]
+        public async Task GetOrganizations()
+        {
+            var userId = "foo";
+            var endpoint = $"https://app.vssps.visualstudio.com/_apis/accounts?memberId={userId}?api-version=5.0-preview.1";
+            var accountsJson = "[{accountId: 'blah', AccountName: 'foo'}, {AccountName: 'foo2'}]";
+
+            var mockClient = new Mock<AdoClient>(null, null);
+
+            mockClient.Setup(x => x.GetAsync(endpoint).Result).Returns(accountsJson);
+
+            using var sut = new AdoApi(mockClient.Object);
+            var result = await sut.GetOrganizations(userId);
+
+            Assert.Equal(2, result.Count());
+            Assert.Contains(result, x => x == "foo");
+            Assert.Contains(result, x => x == "foo2");
+        }
+
+        [Fact]
+        public async Task GetOrganizationId()
+        {
+            var userId = "foo";
+            var adoOrg = "foo-org";
+            var orgId = "blah";
+            var endpoint = $"https://app.vssps.visualstudio.com/_apis/accounts?memberId={userId}&api-version=5.0-preview.1";
+            var accountsJson = "[{accountId: '" + orgId + "', accountName: '" + adoOrg + "'}, {accountName: 'foo2', accountId: 'asdf'}]";
+            var response = JArray.Parse(accountsJson);
+
+            var mockClient = new Mock<AdoClient>(null, null);
+
+            mockClient.Setup(x => x.GetWithPagingAsync(endpoint).Result).Returns(response);
+
+            using var sut = new AdoApi(mockClient.Object);
+            var result = await sut.GetOrganizationId(userId, adoOrg);
+
+            Assert.Equal("blah", result);
+        }
+
+        [Fact]
+        public async Task GetTeamProjects_TwoProjects()
+        {
+            var adoOrg = "foo-org";
+            var teamProject1 = "foo-tp";
+            var teamProject2 = "foo-tp2";
+            var endpoint = $"https://dev.azure.com/{adoOrg}/_apis/projects?api-version=6.1-preview";
+            var json = "[{somethingElse: false, name: '" + teamProject1 + "'}, {id: 'sfasfasdf', name: '" + teamProject2 + "'}]";
+            var response = JArray.Parse(json);
+
+            var mockClient = new Mock<AdoClient>(null, null);
+
+            mockClient.Setup(x => x.GetWithPagingAsync(endpoint).Result).Returns(response);
+
+            using var sut = new AdoApi(mockClient.Object);
+            var result = await sut.GetTeamProjects(adoOrg);
+
+            Assert.Equal(2, result.Count());
+            Assert.Contains(result, x => x == teamProject1);
+            Assert.Contains(result, x => x == teamProject2);
+        }
+
+        [Fact]
+        public async Task GetRepos_ThreeReposOneDisabled()
```

Comment:
nit: Regarding the naming, here a better name would be `GetRepos_Returns_Only_Active_Ones` for example.

Reply:
done